src/controllers/update/remove.cannot.use.controller.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response } from "express";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response) => {
+  return res.render(config.REMOVE_CANNOT_USE_PAGE, {
+    backLinkUrl: config.SOLD_LAND_FILTER_URL,
```

Comment:
back link will need the remove query parameter on it

Reply:
Updated backlink with query parameter and removed update sold land page and replaced it with the remove sold land version.

Reply:
This now looks good to me.